ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -148,10 +182,12 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
       // API handles notebook name collisions and matches the behavior of
       // clicking "new notebook" in the Jupyter UI.
       const workspaceDir = localDir.replace(/^workspaces\//, '');
-      return this.jupyterService.postContents(
+      return this.jupyterService.putContents(
         this.cluster.clusterNamespace, this.cluster.clusterName,
-        workspaceDir, {
-          'type': 'notebook'
+        workspaceDir, this.notebookName + '.ipynb', {
```

Comment:
Currently if they leave newName blank I believe you'll just get a file called ".ipynb".

Reply:
Correct. I can have it check in case and if it is empty make it untitled. That said, Karthik asked me to check if Saturn requires a name, and mirror what they do. So I will check on that

Reply:
Please use "put" if we go this route. Using "untitled" scales to exactly 1 usage, which is the reason I integrated with the Jupyter contents API in the first place.

Reply:
This ends up being obsoleted by it blocking '' as a name option.